src/App.js

```diff
@@ -1,12 +1,60 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
+import axios from 'axios';
 import './App.css';
+import PokemonCard from "./components/PokemonCard";
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+
+    const [allPokemonData, setAllPokemonData] = useState([]);
```

Comment:
Als `pokemon` een object zal worden, dan zou ik een default waarde van `undefined` pakken. Nu lijkt het alsof het een array is en zal blijven, maar dat is niet zo
```suggestion
    const [allPokemonData, setAllPokemonData] = useState();
```